paddle/framework/scope.h

```diff
@@ -88,7 +90,16 @@ class Scope {
             (parent_ && parent_->HasVariable(name)));
   }
 
+  std::string GetVariableName(Variable* const var) const {
```

Comment:
const unordered_map<Variable*, std::string> only can take `Variable* const ` as key type, not `const Variable*`.